### DISCUSSION TITLE
Learncard on React Native

### DISCUSSION THREAD

**Discussion by russelpearl** [learningeconomy/LearnCard#321] [Help] [answered]
Is it possible to use Learncard SDK to react native app?

**Comment by Custard7** [comment, marked as answer]
> Is it possible to use Learncard SDK to react native app?

Yes! While our [UI components are React specific](https://docs.learncard.com/learn-card-sdk/learncard-ux) (not React Native), the rest of the [SDK](https://docs.learncard.com/learn-card-sdk/what-is-learncard) is fully compatible with React Native and can be used to make native mobile apps. 

If you share more about your use case, happy to discuss in more details!

**Reply by russelpearl** [reply]
Thank you!
